script.js

```diff
@@ -0,0 +1,18 @@
+const menuBtn = document.querySelector('.hamburger');
+const menu = document.getElementById('theMenu');
+const xBtn = document.getElementById('xBtn');
+const navLinks = document.querySelectorAll('.navs');
+
+function openMenu() {
+  menu.style.opacity = '90%';
+}
+
+function closeMenu() {
+  menu.style.opacity = 0;
+}
```

Comment:
- [ ] You worked well on the mobile menu feature! 💯 However, be aware that this approach has some issues! Choosing the `opacity` property to show/hide the menu is not the best call, even if the menu has an opacity of 0, it still exists in the DOM and overlays your page. You can evidentiate this by trying to click a button or interact with your page.

A better option would be to toggle between `display: none` and `display: block` or, even better (and kudos if you do it this way) use the `transform: translate` property to implement cool transitions! Please consider applying this change. 🤓 🔧

Reply:
Thanks. I'll implement the transform translate after I do some research on it.

Reply:
Perfect! @ClintonYounge Good luck!